File: easy/challenge_1/solution_1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace daily_programmer
{
    class solution_1
    {
        static void Main(string[] args)
        {
            String name = "", age = "", redditUser = "";
            name = GetInput("Enter your name: ");
            age = GetInput("Enter your age: ");
            redditUser = GetInput("Enter your Reddit username: ");
            Console.Write("Your name is {0}, you are {1} years old, and your username is {2}.", name, age, redditUser);
            Console.Read();
        }

        static String GetInput(string prompt)
        {
            String input = "";
            Console.Write(prompt);
            input = Console.ReadLine();
            return input;
        }
    }
}

File: easy/challenge_1/solution_1.java
package com.daily_programmer;

import java.util.Scanner;

public class Main {

    String name, age, redditun;

    public Main(String name, String age, String redditun){
        this.name = name;
        this.age = age;
        this.redditun = redditun;
    }

    public void displayInfo(){
        System.out.println(String.format("Your name is %s, you are %s years old, and your username is %s."
                , name, age, redditun));
    }

    
    public static void main(String[] args) {
        Main user = new Main(GetInput("Enter your name:"), GetInput("Enter your age:"), GetInput("Enter your username"));
        user.displayInfo();
    }

    public static String GetInput(String prompt){
        Scanner reader = new Scanner(System.in);
        System.out.println(prompt);
        return reader.nextLine();
    }
}
